ruby/ql/lib/codeql/ruby/security/ZipSlipCustomizations.qll

```diff
@@ -0,0 +1,115 @@
+/**
+ * Provides default sources, sinks and sanitizers for reasoning about
+ * zip slip vulnerabilities, as well as extension points for
+ * adding your own.
+ */
+
+private import codeql.ruby.AST
+private import codeql.ruby.ApiGraphs
+private import codeql.ruby.CFG
+private import codeql.ruby.Concepts
+private import codeql.ruby.DataFlow
+private import codeql.ruby.dataflow.BarrierGuards
+private import codeql.ruby.dataflow.RemoteFlowSources
+
+module ZipSlip {
```

Comment:
Nitpick: I'd prefer if all the non-abstract classes in this module (i.e. anything that `extends Sink/Source/Sanitizer`) were `private` to make it easier to change the implementation without changing the public interface.